Readability.js

```diff
@@ -586,7 +586,7 @@ Readability.prototype = {
 
     this._forEachNode(articleContent.children, function (topCandidate) {
       this._cleanMatchedNodes(topCandidate, function (node, matchString) {
-        return /share/.test(matchString) && node.textContent.length < shareElementThreshold;
+        return /[^a-zA-Z0-9]share[^a-zA-Z0-9]/g.test(" " + matchString + " ") && node.textContent.length < shareElementThreshold;
```

Comment:
Could this just be `/\bshare\b/` to look for word breaks? Or does that not work? Also, I assume the regex should be case insensitive - could you add the `i` flag?

Reply:
Good point! `\b` alone won't work because it doesn't check for underscores, but I've tried to simplify this a bit and fix the case issue in 55892033bc5148d1f1ae350b0957136f4c49cab4.